coding/url.cpp

```diff
@@ -174,6 +174,10 @@ string UrlDecode(string_view encodedUrl)
       result += FromHex(encodedUrl.substr(i + 1, 2));
       i += 2;
     }
+    else if (encodedUrl[i] == '+')
```

Comment:
Can we guarantee that UrlDecode is called for queries only? E.g. a + in the path is a valid symbol and should not be converted into a space. Maybe add an explicit param to UrlDecode(..., bool isQuery)?

Reply:
Even Sun Microsystems wasn't concerned about that: https://github.com/openjdk/jdk/blob/aac43184319d852eb792c83dfb52d74a3126108d/src/java.base/share/classes/java/net/URLDecoder.java#L170-L184

Reply:
Don't compare OM with some companies that don't know how to write a good code.

Reply:
This is a good joke.